tests/monitoring/metrics.go

```diff
@@ -141,11 +142,7 @@ func createRunningVM(virtClient kubecli.KubevirtClient) *v1.VirtualMachine {
 	vm, err := virtClient.VirtualMachine(testsuite.GetTestNamespace(vm)).Create(context.Background(), vm, metav1.CreateOptions{})
 	Expect(err).ToNot(HaveOccurred())
 
-	Eventually(func() bool {
-		vm, err := virtClient.VirtualMachine(testsuite.GetTestNamespace(vm)).Get(context.Background(), vm.Name, metav1.GetOptions{})
-		Expect(err).ToNot(HaveOccurred())
-		return vm.Status.Ready
-	}, 300*time.Second, 1*time.Second).Should(BeTrue())
+	Eventually(ThisVM(vm), 300*time.Second, 1*time.Second).Should(BeReady())
```

Comment:
Please use explicit `WithTimeout` and `WithPolling`
```suggestion
	Eventually(ThisVM(vm)).WithTimeout(300*time.Second).WithPolling(time.Second).Should(BeReady())
```

Reply:
done